src/fromager/commands/bootstrap.py

```diff
@@ -72,3 +73,22 @@ def bootstrap(
                 f"removing prebuilt wheel {prebuilt_wheel.name} from download cache"
             )
             filename.unlink()
+
+    constraints_filename = wkctx.work_dir / "constraints.txt"
+    logger.info(f"writing installation dependencies to {constraints_filename}")
+    with open(wkctx.work_dir / "constraints.txt", "w") as f:
+        for name, version in sorted(
+            set(_installation_dependencies(wkctx.all_edges, ""))
+        ):
+            f.write(f"{name}=={version}\n")
+
+
+def _installation_dependencies(
+    all_edges: dict[NormalizedName | str, tuple[str, NormalizedName, str]],
+    name: NormalizedName | str,
+) -> typing.Iterable[tuple[NormalizedName, str]]:
```

Comment:
```suggestion
    name: NormalizedName,
) -> typing.Iterable[tuple[NormalizedName, Version]]:
```

Reply:
I changed the type of the value to Version